pkgs/development/libraries/openssl/default.nix

```diff
@@ -58,6 +59,9 @@ stdenv.mkDerivation rec {
 
     # remove dependency on Perl at runtime
     rm -r $out/etc/ssl/misc $out/bin/c_rehash
+
+    # configure the default trust store
+    ${optionalString (defaultCertificate != null) "ln -s ${defaultCertificate} $out/etc/ssl/cert.pem"}
```

Comment:
Hm, it makes no sense. What I wanted to say is that the symlink add no impurities in the build process because it happens at the end, and does not interfere with the build itself.